docs/content/users/extend/customization-extendibility.md

```diff
@@ -81,7 +81,9 @@ export PATH=$PATH:/var/www/html/somewhereelse/vendor/bin
 
 ## Changing PHP Version
 
-The project's `.ddev/config.yaml` file defines the PHP version to use. The [`php_version`](../configuration/config.md#php_version) can be changed to `5.6`, `7.0`, `7.1`, `7.2`,  `7.3`, `7.4`, `8.0`, `8.1`, `8.2`, '8.3', or `8.4`.
+DDEV keeps up with the "more stable" release, which is one version behind the latest.
```

Comment:
I agree, and why I asked:

> Also, we could add the DDEV PHP version strategy? **Maybe it belongs somewhere else?**

But I do think it would be valuable to include it somewhere in the documentation. If you agree, the question is where then?